### ISSUE TITLE
ANDROID SMAPI ALWAYS CRASHING WHEN I RUN IT!!!!

### ISSUE THREAD

**Issue by Fandeeno207** [ZaneYork/SMAPI-Android-Installer#261]
Just installed stardew Valley from rexdl.com installed SMAPI installer and installed SMAPI Stardew valley and that step is not working. Opening up SMAPI Stardew Valley is impossible because it always crashes whenever I do so.

**Comment by Laithou** [comment]
Oh, really?

**Comment by Fandeeno207** [comment]
> Use 1.4 Stardew Valley, the 1.5 is not supported

Then Why The Absolute FUCK, is "version 1.4 OR LATER is required" written?!?

**Comment by Fandeeno207** [comment]
just tried and this time it's actually giving me a crash detected message. It's saying a lot of stuff but what I could make out of it is "Please ensure the name of the assembly that contains the Type matches the assembly in the full type name: " And then lots of stuff about microsoft.

**Comment by Fandeeno207** [comment]
> 

You are right. but please and excuse my language. But what should I do now?

**Comment by Wonder871** [comment]
Zane York smapi günceleyene kadar bekle pls

**Comment by EdyPlays** [comment]
> Zane York smapi günceleyene kadar bekle pls

İngilizcen çok iyimiş knk hdkshdndd

**Comment by ishammy** [comment]
Stardew Valley 1.5 is different compared to 1.4, because of how much the developer changed in the 1.5 update, what it meant by "version 1.4 OR LATER is required" it just meant updates like bug fixes. The author of SMAPI will make an updated version of SMAPI to make it compatible for 1.5, best you can do now is Wait or downgrade to 1.4 to play with mods

**Comment by Wonder871** [comment]
@EdyPlays nerden anladın türk olduğu mu ?

**Comment by Wonder871** [comment]
Kskskksks

**Comment by Wonder871** [comment]
#246

**Comment by CharlieKitsune** [comment]
> Stardew Valley 1.5 is different compared to 1.4, because of how much the developer changed in the 1.5 update, what it meant by "version 1.4 OR LATER is required" it just meant updates like bug fixes. The author of SMAPI will make an updated version of SMAPI to make it compatible for 1.5, best you can do now is Wait or downgrade to 1.4 to play with mods.
> 
> Hope this helps :)

Hate to be a bother but do you know how to downgrade to 1.4 for mobile? Been trying to find out how and haven’t found much.

**Comment by hotspotbro** [comment]
Brush calm down

**Comment by minacheyo** [comment]
> 

I think, you can download from browser the 1.4 version.
I'm not sure if you can install it on top of 1.5, or you need to uninstall first, so to be sure, just make sure to save your data beforehand!

**Comment by kafkaismommymaterial** [comment]
> 

Hello, so is there any way i can downgrade my save? I am already at year 3 and i do not want to lose it since i already made some good progress, i would be willing to make a new save if i hadn't gotten attached to the world
